### PR TITLE
Count both inbound and outbound requests

### PR DESCRIPTION
**Pull request by StrongMonkey**

### CODE REVIEW

File: pkg/controllers/servicescale/scaler.go [hunk 1]
```diff
@@ -265,6 +265,8 @@ func (s *SimpleScale) scrape() error {
 		metricURL := fmt.Sprintf("http://%s:4191/metrics", pod.Status.PodIP)
 		responseTotalMatchCriteria := fmt.Sprintf("response_total{authority=\"%s-%s.%s.svc.cluster.local", s.app, s.version, s.namespace)
 		requestTotalMatchCriteria := fmt.Sprintf("request_total{authority=\"%s-%s.%s.svc.cluster.local", s.app, s.version, s.namespace)
+		inbound := "direction=\"inbound\""
+		outbound := "direction=\"outbound\""
```
Comment (cbron) [inline pkg/controllers/servicescale/scaler.go]: Nit, but this feels weird. What if you make inbound a `bool` param on `calculateActiveRequests` and if true use the inbound string. Or you could use const.

File: pkg/controllers/servicescale/scaler.go [hunk 2]
```diff
@@ -305,11 +292,32 @@ func (s *SimpleScale) scrape() error {
...
+				response = rqs
+			}
+		}
+		if strings.Contains(scanner.Text(), requestMatch) && strings.Contains(scanner.Text(), direction) {
+			parts := strings.Split(scanner.Text(), " ")
+			if len(parts) == 2 {
+				rqs, _ := strconv.Atoi(parts[1])
+				request = rqs
+			}
+		}
```
Comment (cbron) [inline pkg/controllers/servicescale/scaler.go]: Not very dry, that or I can't spot the difference. Can we abstract into method and pass in ~direction~ match type.

File: pkg/controllers/servicescale/scaler.go [hunk 3]
```diff
@@ -305,11 +292,32 @@ func (s *SimpleScale) scrape() error {
...
+	}
+	return request - response
```
Comment (cbron) [inline pkg/controllers/servicescale/scaler.go:308]: Is there any way this could be negative ?

Comment (StrongMonkey) [reply pkg/controllers/servicescale/scaler.go:308]: don't think so, unless linkerd has it wrong.

Comment (cbron) [reply pkg/controllers/servicescale/scaler.go:308]: Yea an edge case, I'm just wondering if it blips and then records a bunch of closing requests (not sure if thats theoretically possible). If it does that could screw with autoscaling. Could add an absolute call but that is probably premature optimization. We should just keep in mind.

**Review by cbron — APPROVED** [review]
sweet

**Pull request merged**